app/Http/Controllers/ResetPasswordController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\Hash;
+
+use App\Models\Customers;
+use App\Models\VerificationTokens;
+
+class ResetPasswordController extends Controller
+{
+    function __construct()
+    {
+        //set [Customers] as the model/role that will be used for this class
+        \Config::set('auth.providers.users.model', \App\Models\Customers::class);
+    }
+
+    public function resetPassword(Request $request) {
+        $token = $request->token;
+        $oldPassword = $request->old_password;
+        $newPassword = $request->new_password;
```

Comment:
reset password kalau ga salah di check untuk new password and verifikasinya ga sit ??

Reply:
oh iya mas, aku lupa belum bikin validasi new_password + verification_password.
Aku update soon ya.